content/techniques/streaming-files.md

```diff
@@ -39,8 +41,13 @@ import { join } from 'path';
 @Controller('file')
 export class FileController {
   @Get()
+  @Response({ passthrough: true }) res,
```

Comment:
I think you meant to add this as a method argument?

Reply:
Yes, updated